<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AreaEditor</class>
 <widget class="QMainWindow" name="AreaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1023</width>
    <height>768</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Area Editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="areaView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>180</y>
      <width>571</width>
      <height>521</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="areaDataBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>491</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Area Information</string>
    </property>
    <widget class="QLabel" name="widthLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Width:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="heightLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>67</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Height:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="nameIDLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Name ID:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="nameIDBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>37</y>
       <width>381</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="widthBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>77</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="heightBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>117</y>
       <width>251</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>999999999</number>
     </property>
    </widget>
    <widget class="QPushButton" name="resizeButton">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>80</y>
       <width>121</width>
       <height>65</height>
      </rect>
     </property>
     <property name="text">
      <string>Resize</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="editorToolsBox">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>10</y>
      <width>491</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>Editor Tools</string>
    </property>
    <widget class="QLabel" name="letterLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>67</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Sign:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="letterBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>37</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
     <property name="maxLength">
      <number>1</number>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="placeholderText">
      <string/>
     </property>
    </widget>
    <widget class="QCheckBox" name="obstacleBox">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>37</y>
       <width>94</width>
       <height>28</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Obstacle:</string>
     </property>
    </widget>
    <widget class="QLabel" name="objectNameIDLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>67</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Name ID:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QComboBox" name="objectNameIDBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>117</y>
       <width>251</width>
       <height>30</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="colorLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>67</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>Color:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="colorBlueBox">
     <property name="geometry">
      <rect>
       <x>356</x>
       <y>77</y>
       <width>124</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="colorRedBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>77</y>
       <width>124</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="colorGreenBox">
     <property name="geometry">
      <rect>
       <x>223</x>
       <y>77</y>
       <width>124</width>
       <height>31</height>
      </rect>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="maximum">
      <number>255</number>
     </property>
     <property name="value">
      <number>255</number>
     </property>
    </widget>
    <widget class="QPushButton" name="copyButton">
     <property name="geometry">
      <rect>
       <x>357</x>
       <y>116</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Copy tile</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="objectInfoBox">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>180</y>
      <width>421</width>
      <height>521</height>
     </rect>
    </property>
    <property name="title">
     <string>Object Info</string>
    </property>
    <widget class="QStackedWidget" name="objectInfoWidget">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>381</width>
       <height>481</height>
      </rect>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="empty"/>
     <widget class="QWidget" name="item">
      <widget class="QLabel" name="itemResourceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>101</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Item resource:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="itemResourceBox">
       <property name="geometry">
        <rect>
         <x>130</x>
         <y>7</y>
         <width>251</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="npc">
      <widget class="QLabel" name="npcResourceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>NPC resource:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="npcResourceBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>7</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="dialogResourceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>50</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Dialog resource:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="dialogResourceBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>47</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="storeResourceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>90</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Store resource:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="storeResourceBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>87</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QComboBox" name="allegianceBox">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>127</y>
         <width>231</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="allegianceLabel">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Allegiance:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="door">
      <widget class="QCheckBox" name="doorOrientationBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>10</y>
         <width>361</width>
         <height>28</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Horizontal orientation:</string>
       </property>
      </widget>
      <widget class="QCheckBox" name="lockedBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>50</y>
         <width>361</width>
         <height>28</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Locked:</string>
       </property>
      </widget>
      <widget class="QSpinBox" name="doorGreenBox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>127</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="doorBlueBox">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>127</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
      <widget class="QLabel" name="doorColorLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>130</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Color:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QSpinBox" name="doorRedBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>127</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
      <widget class="QCheckBox" name="openBox">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>361</width>
         <height>28</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string>Open:</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="sign">
      <widget class="QLabel" name="dialogIDLabel">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>10</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Dialog ID:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="signLetterLabel">
       <property name="geometry">
        <rect>
         <x>-10</x>
         <y>50</y>
         <width>81</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Sign:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QLineEdit" name="signLetterBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>47</y>
         <width>291</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
      <widget class="QSpinBox" name="signGreenBox">
       <property name="geometry">
        <rect>
         <x>190</x>
         <y>87</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="signBlueBox">
       <property name="geometry">
        <rect>
         <x>290</x>
         <y>87</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
      <widget class="QSpinBox" name="signRedBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>87</y>
         <width>91</width>
         <height>31</height>
        </rect>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
       <property name="maximum">
        <number>255</number>
       </property>
       <property name="value">
        <number>255</number>
       </property>
      </widget>
      <widget class="QLabel" name="signColorLabel">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>71</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Color:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QComboBox" name="dialogIDBox">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>7</y>
         <width>291</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1023</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Plik</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>true</bool>
   </property>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>Otwórz</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Zapisz</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="text">
    <string>Zapisz jako</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Zakończ</string>
   </property>
  </action>
 </widget>
 <tabstops>
  <tabstop>nameIDBox</tabstop>
  <tabstop>widthBox</tabstop>
  <tabstop>heightBox</tabstop>
  <tabstop>areaView</tabstop>
  <tabstop>resizeButton</tabstop>
  <tabstop>letterBox</tabstop>
  <tabstop>obstacleBox</tabstop>
  <tabstop>colorRedBox</tabstop>
  <tabstop>colorGreenBox</tabstop>
  <tabstop>colorBlueBox</tabstop>
  <tabstop>objectNameIDBox</tabstop>
  <tabstop>copyButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
